ui/apps/platform/src/Containers/Collections/collections.utils.ts

```diff
@@ -0,0 +1,41 @@
+export type CollectionPageAction =
+    | {
+          type: 'create';
+      }
+    | {
+          type: 'edit' | 'clone' | 'view';
```

Comment:
Note that `"view"` is implicit in the URL for collection pages, but is included here for a couple of reasons:

- Clarity when reading/writing code, instead of having to work with `'edit' | 'clone' | undefined`.
- Better type narrowing, since TypeScript can't use the type of a property to narrow in a discriminated union (TIL!)

Both `/collections/<collection_id>` and `/collections/<collection_id>?action=view` are technically valid, but when creating links we should prefer the former.